MassEffectModder/MassEffectModder/MassEffectModder.pro

```diff
@@ -171,7 +171,9 @@ win32-g++: {
     # Disable compiler warning
     QMAKE_CXXFLAGS += -Wno-deprecated-copy
     QMAKE_LFLAGS_RELEASE = "-Wl,--relax -Wl,--disable-dynamicbase"
+    # QMAKE_LFLAGS_RELEASE = "-Wl,--relax" # Building on Windows Qt seems to use this version
```

Comment:
what is about this commented line?

Reply:
This is added so I could compile on Windows, but I commented it out on PR so it would not change your builds or releases. I will see if maybe updating my local Qt will fix this when building on Windows.